### ISSUE TITLE
number type gives validation error

### ISSUE THREAD

**Issue by bombompb** [makinacorpus/angular2-schema-form#34]
I have the following simple schema:
```json
{
	"type": "object",
	"properties": {
		"maxNumberOfRetries": {
			"type": "number",
			"minimum": 0,
			"maximum": 10,
			"description": "Maximum number of retries"
		}
	}
}
```

This results in a nice numeric field in the UI with up/down spinner buttons.

However when I subscribe to the validation errors like this:

```
this.subscriptionForErrors = this.schemaForm.rootProperty.errorsChanges.subscribe(e => {
        if(e != null) {
          console.log('errors:');
           e.forEach(ee => console.log(ee));
        }
      });
```

I get the following error logged when I enter a number in the UI field:
code: INVALID_TYPE
message: Expected type number but found type string

Any idea?

**Comment by sithwin** [comment]
It is still returning **string**. Pleas check again!
`{
  "student": {
    "lastName": "",
    "firstName": "",
    "id": "**222**",
    "dateOfBirth": ""
  }`

**Comment by ebrehault** [comment]
I cannot reproduce the error since my last fix.
Maybe you are not using the GitHub master version?
(anyway I have just released 1.0.0-alpha.32 which contains this fix)

**Comment by sithwin** [comment]
It is fixed after I updated the released 1.0.0-alpha.32. 

Thanks!
